src/porepy/models/momentum_balance.py

```diff
@@ -671,11 +623,6 @@ class SolutionStrategyMomentumBalance(pp.SolutionStrategy):
 
     """
 
-    nd: int
-    """Ambient dimension of the problem. Normally set by a mixin instance of
-    :class:`porepy.models.geometry.ModelGeometry`.
-
-    """
     solid: pp.SolidConstants
```

Comment:
Are not solid and EquationSystem defined in the protocol?

Reply:
Removed